src/collections/_documentation/workflow/search.md

```diff
@@ -3,7 +3,17 @@ title: Search
 sidebar_order: 3
 ---
 
-Search in Sentry currently exists in a limited fashion. You can perform basic CONTAINS matches on the title of an issue or exact matches on tags.
+Search is available on several major Sentry views: Issues, Events, and Releases.
+
+{% capture __alert_content -%}
+Discover is Sentry's query builder for aggregating raw event data, and has its own unique syntax not covered here. For more information, see Discover.
```

Comment:
Can we highlight that Discover is better, and link to it?

Reply:
```suggestion
Discover is Sentry's query builder for aggregating raw event data and has its own unique syntax not covered here. For more information, see full Discover documentation.
```

Reply:
Hi @eurica, I'm not sure we can link inside of an alert box.